### PR TITLE
feat: The new verifier now works on windows too

### PR DESCRIPTION
**Pull request by TimothyJones**
This was a bit of an ordeal. I refactored a bit along the way.

I think we should squash merge this one

### CODE REVIEW

File: src/pact-standalone.spec.ts [hunk 1]
```diff
@@ -37,146 +37,154 @@ describe('Pact Standalone', function () {
   });
 
   describe('Check if OS specific files are there', () => {
-    describe('OSX', () => {
-      beforeEach(() => (pact = standalone('darwin')));
+    if (!process.env.ONLY_DOWNLOAD_PACT_FOR_WINDOWS) {
```
Comment (mefellows) [inline src/pact-standalone.spec.ts:40]: 👌

**Comment by TimothyJones** [comment]
Also the appveyor build is a red herring

File: script/lib/robust-bash.sh [hunk 2]
```diff
@@ -10,11 +10,18 @@ if [ -z "${LIB_ROBUST_BASH_SH:-}" ]; then
     echo "🔵  ${1:-}"
   }
 
+  function debug_log {
```
Comment (mefellows) [inline script/lib/robust-bash.sh:13]: moar emoji!

**Review by mefellows — APPROVED** [review]
Awesome Tim. Skipping the bits on Windows that don't work makes sense to me.

We'll probably need some integration test of sorts at some point to ensure that the released artefact actually works on Windows. But, one problem at a time...

File: script/lib/robust-bash.sh (hunk 2 above)
Comment (TimothyJones) [reply script/lib/robust-bash.sh:13]: 👀

File: src/pact-standalone.spec.ts (hunk 1 above)
Comment (TimothyJones) [reply src/pact-standalone.spec.ts:40]: Haha! I'd like not to have to do this. But it gets it going for now.

**Pull request merged**